### ISSUE TITLE
Can it not highlight anything and just have a popover in the center of the screen?

### ISSUE THREAD

**Issue by sumtsui** [kamranahmedse/driver.js#197]
I am developing for an intro that some steps don't highlight anything, but just have a dialog in the center of screen. Cannot have any existing api to achieve that.

**Comment by vitorleonel** [comment]
@sumtsui I'm simulate this with custom class and set position *mid-center*.  It would be nice if this were native.

**Comment by etiennebacher** [comment]
@vitorleonel Thank you for this idea, but the whole body is highlighted in white behind the popover. I know I need to add `opacity: 0` to `driver-highlighted-element-stage` but only for this popover, not for the others, so that the other elements are still highlighted when it comes to them. Could you share the code for your solution?

Here's what I've done so far:
```
<!DOCTYPE html>
<html lang="en">
  <head>
      <!-- Files for driver.js --> 
      <script src="https://unpkg.com/driver.js/dist/driver.min.js"></script>
      <link rel="stylesheet" href="https://unpkg.com/driver.js/dist/driver.min.css">
  </head>

  <body>

    <button id="btn">launch driver.js</button>
    <p id = "hi">Hello</p>

      <script>
          const driver = new Driver();
          
          driver.defineSteps([
          {
            element: 'body',
            popover: {
              title: 'Test 1',
              position: 'mid-center',
              className: "fooo"
            }
           },
            {
            element: '#hi',
            popover: {
              title: 'Test 2',
              position: 'bottom'
            }
           }
         ]);
         let btn = document.querySelector('#btn');
         btn.addEventListener('click', function(){
          event.stopPropagation()
          driver.start();
         });
         </script>
      
  </body>
</html>
```
If I add `opacity: 0`, then the text in the second popover will not be highlighted. I don't know how to use the custom class name to fix this.

**Comment by etiennebacher** [comment]
Nevermind, just saw [this answer](https://github.com/kamranahmedse/driver.js/issues/67#issuecomment-784225141) in #67

**Comment by kamranahmedse** [comment]
We just released [v1.0](https://github.com/kamranahmedse/driver.js/releases/tag/1.0.3) of driver.js which has been rewritten from the ground up. It is now possible. Please have a [look at the docs](https://driverjs.com/) and do let me know if you face any issues.
